crates/iota-json-rpc-tests/tests/read_api.rs

```diff
@@ -1487,104 +1487,108 @@ async fn try_get_past_object_version_not_found() {
     assert!(at_least_one_version_not_found)
 }
 
-#[sim_test]
-async fn try_get_past_object_deleted() {
-    let cluster = TestClusterBuilder::new().build().await;
-    let http_client = cluster.rpc_client();
-    let address = cluster.get_address_0();
-
-    let objects = cluster
-        .get_owned_objects(address, Some(IotaObjectDataOptions::full_content()))
-        .await
-        .unwrap();
+mod example {
```

Comment:
I understand that the classification of these tests into the `example` module stems from the need to query them in the CI.

However, when I read the test module as a maintainer and see the nested `example` module I am rather confused. So I would suggest renaming that something that is more fit to the distinguishing characteristics of these tests w.r.t. to the rest in `super`. 

Perhaps `mod move` is the best candidate.

```suggestion
mod move {
```

Reply:
That's fine with me 👍

Reply:
`move` is a keyword, so I used `move_tests` instead

Reply:
perfect, thanks!